lang/src/test/kotlin/org/partiql/lang/eval/evaluatortestframework/ExprValueStrictEqualsTest.kt

```diff
@@ -0,0 +1,153 @@
+package org.partiql.lang.eval.evaluatortestframework
+
+import com.amazon.ion.system.IonSystemBuilder
+import org.junit.Assert
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ArgumentsSource
+import org.partiql.lang.CompilerPipeline
+import org.partiql.lang.eval.EvaluationSession
+import org.partiql.lang.util.ArgumentsProviderBase
+
+class ExprValueStrictEqualsTest {
+    data class TestCase(val q1: String, val q2: String)
```

Comment:
I think this tests collection will read more convincingly if equality and inequality tests where coming together for each expression kind.